skins/laika/src/components/pages/membership_confirmation/MembershipSummary.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+	<div class="donation-summary">
+		<div class="payment-summary" v-html="getSummary()"></div>
+		<div class="payment-email" v-html="getEmail()"></div>
+		<div class="has-margin-top-18">{{ $t( 'membership_confirmation_success_text' ) }}</div>
+	</div>
+</template>
+
+<script lang="js">
+import Vue from 'vue';
+import { AddressTypeModel, addressTypeName } from '@/view_models/AddressTypeModel';
+
+class PrivateApplicantRenderer {
+	static renderAddress( address, country ) {
+		return address.salutation + ' ' + address.fullName + ', '
+				+ address.streetAddress + ', ' + address.postalCode + ' ' + address.city + ', ' + country;
+	}
+}
+class CompanyApplicantRenderer {
+	static renderAddress( address, country ) {
+		return address.fullName + ', '
+				+ address.streetAddress + ', ' + address.postalCode + ' ' + address.city + ', ' + country;
+	}
+}
+
+const addressTypeRenderers = {
+	[ addressTypeName( AddressTypeModel.PERSON ) ]: PrivateApplicantRenderer,
+	[ addressTypeName( AddressTypeModel.COMPANY ) ]: CompanyApplicantRenderer,
+};
+
+export default Vue.extend( {
+	name: 'MembershipSummary',
+	props: [
+		'address',
+		'addressType',
```

Comment:
Looks like `addressType` is now part of `address` so this property needs to be removed from here.

Reply:
Good catch